Structure_AdapterModules/ModuleLoader.cs

```diff
@@ -40,6 +41,10 @@ public static void LoadModules(this BHoMAdapter adapter)
             adapter.AdapterModules.Add(new CopyNodeProperties());
             adapter.AdapterModules.Add(new GetCasesFromCombinations());
             adapter.AdapterModules.Add(new GetCombinationsFromCombinations());
+            adapter.AdapterModules.Add(new GetLoadElementsWithoutID<Bar>(adapter));
+            adapter.AdapterModules.Add(new GetLoadElementsWithoutID<Node>(adapter));
+            adapter.AdapterModules.Add(new GetLoadElementsWithoutID<IAreaElement>(adapter));
+            adapter.AdapterModules.Add(new GetLoadElementsWithoutID<BHoMObject>(adapter));  //Gravity loads
```

Comment:
Could you extend the "Gravity loads" comment elaborating a bit? Not too transparent to me